lib/mongo/bulk_write.rb

```diff
@@ -217,14 +217,40 @@ def insert_one(documents, server, operation_id, session, txn_num)
     end
 
     def update_one(documents, server, operation_id, session, txn_num)
+      if server.max_wire_version < 5 && hint_provided?(documents)
+        raise Error::UnsupportedCommandOption.new(
+          option_name: 'hint',
+          command_name: "bulk write with #{__callee__}",
```

Comment:
My web search for "ruby __callee__" produced no results, can you please link to documentation for it?

Reply:
https://apidock.com/ruby/Kernel/__callee__